idflakies-core/pom.xml

```diff
@@ -0,0 +1,169 @@
+        <groupId>edu.illinois.cs</groupId>
+        <artifactId>idflakies</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>idflakies-core</artifactId>
+    <packaging>jar</packaging>
+
+    <name>iDFlakies Core</name>
+    <description>Core logic for iDFlakies</description>
+
+    <developers>
+        <developer>
+            <name>Wing Lam</name>
+            <email>winglam2@illinois.edu</email>
+        </developer>
+        <developer>
+            <name>Reed Oei</name>
+            <email>reedoei2@illinois.edu</email>
+        </developer>
+        <developer>
+            <name>August Shi</name>
+            <email>awshi2@illinois.edu</email>
+        </developer>
+    </developers>
+
+    <licenses>
+        <license>
+            <name>University of Illinois/NCSA Open Source License</name>
+            <url>https://opensource.org/licenses/NCSA</url>
+        </license>
+    </licenses>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>edu.illinois.cs</groupId>
+            <artifactId>testrunner-maven-plugin</artifactId>
+            <version>1.1</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.maven.shared</groupId>
+            <artifactId>maven-invoker</artifactId>
+            <version>3.0.1</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.opencsv</groupId>
+            <artifactId>opencsv</artifactId>
+            <version>4.3</version>
+        </dependency>
+        <dependency>
+            <groupId>com.reedoei</groupId>
+            <artifactId>eunomia</artifactId>
+            <version>1.4.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.5</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>25.1-jre</version>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+        </dependency>
+        <dependency>
+            <groupId>org.dom4j</groupId>
+            <artifactId>dom4j</artifactId>
+            <version>2.1.1</version>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.xerial/sqlite-jdbc -->
```

Comment:
I think we can just remove this comment (was probably part of what one copies from mvn central).